release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3354,6 +3354,59 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.16.39
+[id="ocp-4-16-39_{context}"]
+=== RHSA-2025:4008 - {product-title} {product-version}.39 bug fix and security update
+
+Issued: 23 April 2025
+
+{product-title} release {product-version}.39 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:4008[RHSA-2025:4008] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:4010[RHBA-2025:4010] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.39 --pullspecs
+----
+
+[id="ocp-4-16-39-known-issues_{context}"]
+==== Known issues
+ * IPsec is not supported on {op-system-base-full} compute nodes because of a `libreswan` incompatiblility issue between a host and an `ovn-ipsec` container that exists in each compute node. (link:https://issues.redhat.com/browse/OCPBUGS-42952[OCPBUGS-42952]). 
```

Comment:
```suggestion
 * IPsec is not supported on {op-system-base-full} compute nodes because of a `libreswan` incompatiblility issue between a host and an `ovn-ipsec` container that exists in each compute node. (link:https://issues.redhat.com/browse/OCPBUGS-52952[OCPBUGS-52952]). 
```